crates/starknet/src/lib.rs

```diff
@@ -221,6 +221,11 @@ impl Starknet {
 
         Ok(())
     }
+
+    pub fn block_number(&self) -> DevnetResult<BlockNumber> {
+        let block_len = self.blocks.num_to_block.len();
```

Comment:
not sure btw here, will this case be always >0, because of the genesis block, and do we need to handle some edge cases around that

Reply:
This is only good under two conditions:
- there is no genesis block with number 0
- the first mined block has number 1

A more long term approach would b to get the last block and extract the number from it. And if there is no block, perhaps we can raise an error or return -1.